source/_integrations/deako.markdown

```diff
@@ -4,7 +4,7 @@ description: Instructions on how to integrate Deako Smart Lighting into Home Ass
 ha_category:
   - Switch
 ha_iot_class: Local Polling
-ha_release: 2024.10
+ha_release: "2024.10"
```

Comment:
is this needed? most files don't use double quotes here

Reply:
Otherwise it's interpreted as `2024.1`